src/analytics/index.tsx

```diff
@@ -52,3 +52,8 @@ export const sendAnalyticsEvent: typeof sendAnalyticsTraceEvent = (event, proper
     sendAnalyticsTraceEvent(event, properties)
   }
 }
+
+// This is only used for initial page load so we can get the user's country
+export const sendAnalyticsEventIgnoreTracking: typeof sendAnalyticsTraceEvent = (event, properties) => {
+  sendAnalyticsTraceEvent(event, properties)
```

Comment:
how is this function ignoring tracking? it's just calling the logging function in the same way as `sendAnalyticsEvent` right?

Reply:
`sendAnalyticsEvent` checks to make sure the analytics ignore key is not set - this does not

Reply:
^ probably worth putting this as a comment in code